Gemfile

```diff
@@ -38,3 +38,4 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+gem 'rubocop', require: false
```

Comment:
Add required version like this `, '~> 1.18'` and consider moving this entry to particular `group` - I guess the best choice will be `group :development` or `group :development, :test do`. If you adding gems out of group it means that those gems will be installed for all environments - for instance for production is kinda useless 🤷